lib/mollie/chargeback.rb

```diff
@@ -4,33 +4,30 @@ class Chargeback < Base
     attr_accessor :id,
                   :payment,
                   :amount,
-                  :chargeback_datetime,
+                  :settlement_amount,
+                  :created_at,
                   :settlement_id,
-                  :reversed_datetime
-
+                  :payment_id,
+                  :reversed_at
```

Comment:
The API response includes a `_link` node, which can be used to access related payments.

Can we:
1. Add an alias from `links` to `_links`
2. Allow retrieving the corresponding payment by introducing the helper method `payment`